clifford/test/test_dpga.py

```diff
@@ -128,7 +133,7 @@ def test_line(self):
 
             # Lines are invariant to screw transformations about their axis
             axis = p1vec - p2vec
-            rotation_biv = axis[0]*(e23 - e67) + axis[1]*(e57 - e13) + axis[2]*(e12 - e56)
+            rotation_biv = axis[0]*(e23 - e2b3b) + axis[1]*(e1b3b - e13) + axis[2]*(e12 - e1b2b)
```

Comment:
Note this is actually much easier to read :)

Reply:
Would be neat if the blades were something like `e1 ^ e2 ^ e3b ^ e4b` == `e12_34b`, but that's a little more effort than I can be bothered to spend.

Reply:
Is it the case that `e23 - e2b3b == w2 ^ w3` or something?

Reply:
maybe, can't remember any more, would have to look at the paper again